libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -655,7 +655,7 @@ string YulUtilFunctions::arrayConvertLengthToSize(ArrayType const& _type)
 						</byteArray>
 					})")
 					("functionName", functionName)
-					("elementSize", _type.location() == DataLocation::Memory ? baseType.memoryHeadSize() : baseType.calldataEncodedSize())
+					("elementSize", to_string(_type.location() == DataLocation::Memory ? baseType.memoryHeadSize() : baseType.calldataEncodedSize()))
```

Comment:
Needs rebase?

Reply:
Well, I would trust that git merge can handle that without conflicts, but yeah, I'll rebase :)